eng/pipelines/common/templates/steps/publish-test-results-step.yml

```diff
@@ -29,7 +32,7 @@ steps:
     testResultsFormat: VSTest
     mergeTestResults: true
     buildPlatform: '${{parameters.platform }}'
-    buildConfiguration: '${{parameters.configuration }}'
+    buildConfiguration: '${{parameters.buildConfiguration }}'
     ${{ if eq(parameters.operatingSystem, 'Windows') }}:
       testResultsFiles: 'TestResults/*.trx'
       testRunTitle: 'Windows Tests'
```

Comment:
It's not related specifically to running tests in Debug, but while we're looking at the testing infrastructure: I've found that a hardcoded test run title makes the Pipelines test results hard to filter through - if something is failing on one or two legs, it's hard to tell which leg has caused the failure because everything is grouped under either "Windows Tests" or "Linux Tests".

Could we change the test run titles to align with the stage titles (Win22_Sql19_x86, Ubuntu20_Azure_Sql, etc.) and job names (net8_0_AnyCPU_NativeSNI_1) please?

Reply:
The step names and the job names are fine. If you navigate to the Tests tab though, there's a list of tests grouped by the test run title ("Windows Tests" and "Linux Tests"). The items in this list don't identify which job caused the failure.

<img width="335" height="506" alt="image" src="https://github.com/user-attachments/assets/6de18084-5943-4ce8-8ed2-b76824f1503a" />